gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -2098,16 +2099,26 @@ public GoogleCloudStorageItemInfo composeObjects(
 
   <RequestT extends StorageRequest<?>> RequestT initializeRequest(
       RequestT request, String bucketName) throws IOException {
+    String accessToken = null;
     if (storageRequestAuthorizer != null) {
-      storageRequestAuthorizer.authorize(request);
+      try {
+        accessToken = storageRequestAuthorizer.authorize(request);
+      } catch (UnsupportedOperationException e) {
```

Comment:
`storageRequestAuthorizer` throws `AccessDeniedException` , I do not see any references to `UnsupportedOperationException` as a checked Exception from the `authorize` method call.

Do we need to catch this exception here ?